storage/txcache/config.go

```diff
@@ -46,17 +51,77 @@ func (config *CacheConfig) verify() error {
 			return fmt.Errorf("%w: config.CountThreshold is invalid", errInvalidCacheConfig)
 		}
 
-		if config.NumSendersToEvictInOneStep == 0 {
-			return fmt.Errorf("%w: config.NumSendersToEvictInOneStep is invalid", errInvalidCacheConfig)
+		if config.NumSendersToPreemptivelyEvict == 0 {
+			return fmt.Errorf("%w: config.NumSendersToPreemptivelyEvict is invalid", errInvalidCacheConfig)
 		}
 	}
 
 	return nil
 }
 
-func (config *CacheConfig) getSenderConstraints() senderConstraints {
+func (config *ConfigSourceMe) getSenderConstraints() senderConstraints {
 	return senderConstraints{
 		maxNumBytes: config.NumBytesPerSenderThreshold,
 		maxNumTxs:   config.CountPerSenderThreshold,
 	}
 }
+
+func (config *ConfigSourceMe) String() string {
```

Comment:
comment exported function. add treating of error.

Reply:
Fixed.